maskrcnn_benchmark/data/datasets/coco.py

```diff
@@ -79,9 +79,11 @@ def __getitem__(self, idx):
         classes = torch.tensor(classes)
         target.add_field("labels", classes)
 
-        masks = [obj["segmentation"] for obj in anno]
-        masks = SegmentationMask(masks, img.size, mode='poly')
-        target.add_field("masks", masks)
+        masks = [obj["segmentation"] for obj in anno if any(obj["segmentation"])]
```

Comment:
maybe you could just add something similar to what is done for keypoints just below?
```
if anno and 'segmentation' in anno[0]:
    # rest of the code here
```

Reply:
Hi, 

Thanks for this suggestion, I have modified the code as the keypoint part. Please add more comments if further modification is required.